src/features/targetedOutreach/hooks/useShowOutreachPopup.tsx

```diff
@@ -0,0 +1,26 @@
+import useIsSafeOwner from '@/hooks/useIsSafeOwner'
+import { MAX_ASK_AGAIN_DELAY } from '@/features/targetedOutreach/constants'
+import { useAppSelector } from '@/store'
+import { selectCookieBanner } from '@/store/popupSlice'
+import type { getSubmission } from '@safe-global/safe-client-gateway-sdk'
+
+export const useShowOutreachPopup = (
+  isDismissed: boolean | undefined,
+  askAgainLaterTimestamp: number | undefined,
+  submission: Awaited<ReturnType<typeof getSubmission>> | undefined,
+) => {
+  const cookiesPopup = useAppSelector(selectCookieBanner)
+  const isSigner = useIsSafeOwner()
+
+  const isTargetedSafe = submission?.targetedSafeId && !submission.completionDate
```

Comment:
If its not a targeted safe, `submission` should be undefined. Would it also work to just check for that instead of `targetedSafeId` and `completionDate`?

Reply:
It needs to check completion date to make sure they have not already done the survey. So the name is not quite right

maybe this instead?
```
  const isTargetedSafe = !!submission
  const hasCompletedSurvey =  !submission?.completionDate
```

Reply:
Thats more clear thanks!